test/e2e/generator/generate.go

```diff
@@ -196,6 +206,14 @@ func generateNode(r *rand.Rand, mode e2e.Mode, startAt int64, forceArchive bool)
 		node.SnapshotInterval = 3
 	}
 
+	if node.Mode == "validator" && node.PrivvalProtocol == "file" {
```

Comment:
Instead of checking for `PrivvalProtocol == "file"`, we should force-set it to `file` when a misbehavior is enabled. Otherwise, the effective probability will be the product, i.e. 3% for validators only, which is very low.

Reply:
Okay I wasn't sure which probability we wanted to dominate

Reply:
Yeah - I think `file` already gets pretty good coverage, since it has a high probability to begin with.